public/redux/reducers/elasticsearch.ts

```diff
@@ -93,7 +94,7 @@ const reducer = handleActions<ElasticsearchState>(
       ): ElasticsearchState => ({
         ...state,
         requesting: false,
-        errorMessage: action.error.data.error,
+        errorMessage: get(action, 'error.data.error', action.error),
```

Comment:
this looks like this would return `action.error.data.error` if exists, or `action.error` if not. Wouldn't it always be one or the other?

Reply:
Not sure about it, haven't done all error cases. To keep compatible with original cases and handle remote indices error, it's safe to fall back to action.error as action is not undefined. There are other places where error handling use different ways, created an issue to track this problem and fix it later.https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/issues/245